### PR TITLE
make all code that relied on fieldname_en fields only tap into fielname again

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/6063

Tests, interestingly, pass with only the smallest of changes

### CODE REVIEW

File: network-api/networkapi/wagtailcustomization/static/wagtail_modeltranslation/js/language_toggles.js [hunk 1]
```diff
@@ -1,179 +1,192 @@
...
-
-    const tabbedContent = $(`form .tab-content`);
-    const topLevel = (tabbedContent.length > 0) ? tabbedContent.first() : $(`.content form`);
-    const languageCodeRegex = new RegExp(' \\[('+wagtailModelTranslations.languages.join('|')+')\\]');
-
-    if (topLevel.length === 0) {
-      // obviously, if we don't have an element to attach
-      // the picker to, we might as well stop right now.
-      return;
+jQuery(() => {
```
Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/static/wagtail_modeltranslation/js/language_toggles.js:2]: this appears to be `prettier`

File: network-api/networkapi/wagtailcustomization/static/wagtailadmin/js/copy-helper.js [hunk 2]
```diff
@@ -38,12 +38,16 @@ sync.addEventListener(`click`, () => {
 // there any errors? If so, we need to immediately reveal all fields.
 if (document.querySelectorAll(`p.error-message`).length !== 0) {
   showAll.click();
-  document.dispatchEvent(new CustomEvent(`wagtailcustomization:copy-helper:done`));
+  document.dispatchEvent(
```
Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/static/wagtailadmin/js/copy-helper.js:41]: more `prettier`

File: source/js/buyers-guide/components/creep-vote/creep-vote.jsx [hunk 3]
```diff
@@ -76,7 +76,7 @@ export default class CreepVote extends Component {
 
     // If we're running for the new wagtail version of PNI, voting is a post to the product's URL instead.
     if (document.getElementById("product-research").dataset.isWagtailPage) {
-        url = ".";
+      url = ".";
```
Comment (Pomax) [inline source/js/buyers-guide/components/creep-vote/creep-vote.jsx:79]: more `prettier`

File: source/js/foundation/inject-react/multipage-nav.js [hunk 4]
```diff
@@ -9,18 +9,16 @@ export default (apps) => {
   const targetNode = document.querySelector(
     `#multipage-nav-mobile .container .row .col-12`
   );
-  const multipageLinks = document.querySelectorAll(`#multipage-nav a`)
+  const multipageLinks = document.querySelectorAll(`#multipage-nav a`);
```
Comment (Pomax) [inline source/js/foundation/inject-react/multipage-nav.js:12]: and more `prettier`

**Review by KalobTaulien — APPROVED** [review]
Nice! Everything works perfectly 👌

**Pull request merged**